From 8d5178f7b7b42da594c4e00ce635182735bc5258 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Sun, 4 Feb 2018 04:01:51 -0500
Subject: [PATCH] Add default bootargs

---
Signed-off-by: Khem Raj <raj.khem@gmail.com>
Upstream-Status: Pending

Index: git/include/configs/meson-gx-common.h
===================================================================
--- git.orig/include/configs/meson-gx-common.h
+++ git/include/configs/meson-gx-common.h
@@ -40,6 +40,7 @@
 #include <config_distro_bootcmd.h>
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
+	"bootargs=console=ttyAML0,115200 rootfstype=ext4 rootwait\0" \
 	"fdt_addr_r=0x01000000\0" \
 	"scriptaddr=0x1f000000\0" \
 	"kernel_addr_r=0x01080000\0" \
